docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+# Support Bundle User Guide
```

Comment:
I suppose we need link this file to somewhere to let user know about this new user guide?
If there is no proper place, you may add a line the `Collecting support information` section to let user know about other way to collect support bundle.